muikku-core-plugins/src/main/resources/fi/otavanopisto/muikku/plugins/workspace/WorkspaceJsPluginMessages_fi.properties

```diff
@@ -268,17 +268,36 @@ plugin.workspace.journal.deleteEntry.dialog.cancelButton = Peruuta
 plugin.workspace.journal.deleteEntry.dialog.description = Oletko varma ett� haluat poistaa p�iv�kirjamerkinn�n?
 plugin.workspace.journal.newEntry.title = Uusi p�iv�kirjamerkint�
 plugin.workspace.journal.editEntry.title = Muokkaa p�iv�kirjamerkint��
+plugin.workspace.journal.comments.title = Kommentit
+plugin.workspace.journal.newComment.title = Uusi kommentti
+plugin.workspace.journal.editComment.title = Muokkaa kommenttia
+plugin.workspace.journal.entry.comments.title = Kommentit
+plugin.workspace.journal.newCommentButton.label = Uusi kommentti
+plugin.workspace.journal.editCommentButton.label = Muokkaa
+plugin.workspace.journal.deleteCommentButton.label = Poista
 plugin.workspace.journal.entry.title.label = Otsikko
```

Comment:
Lines not matching up with en locale (just a heads up because this just makes it more difficult in the future to keep track of the locales)

Reply:
Should be fixed now